integration-tests/src/test/java/oracle/kubernetes/operator/utils/RcuSecret.java

```diff
@@ -56,14 +61,101 @@ public RcuSecret(
     ExecResult result = TestUtils.exec(command);
     LoggerHelper.getLocal().log(Level.INFO, "command result " + result.stdout().trim());
   }
+  
+  /**
+   * Construct RCU secret.
+   * 
+   * @param secretname secret name
+   * @param rcuPrefix username
+   * @param password password
+   * @param sysUsername sys username
+   * @param sysPassword sys password
+   * @throws Exception on failure
+   */
+  public RcuSecret(
+      String namespace,
+      String secretName,
+      String rcuPrefix,
+      String rcuSchemaPass,
+      String rcuConnection
+  )
+      throws Exception {
+    this.namespace = namespace;
+    this.secretName = secretName;
+    this.rcuPrefix = rcuPrefix;
+    this.rcuSchemaPass = rcuSchemaPass;
+    this.rcuConnection = rcuConnection;
+    
+    // delete the secret first if exists
+    deleteSecret();
 
+    // create the secret
+    String command =
+        "kubectl -n "
+            + this.namespace
+            + " create secret generic "
+            + this.secretName
+            + " --from-literal=rcu_prefix="
+            + this.rcuPrefix
+            + " --from-literal=rcu_schema_password="
+            + this.rcuSchemaPass
+            + " --from-literal=rcu_db_conn_string="
+            + this.rcuConnection;
+            
+    LoggerHelper.getLocal().log(Level.INFO, "Running " + command);
+    
+    try {
+      ExecResult result = TestUtils.exec(command);
+      LoggerHelper.getLocal().log(Level.INFO, "command result " + result.stdout().trim());
+    } catch (Exception ex) {
+      ex.getCause();
```

Comment:
what will this do? nothing? swallow the exception? why?

Reply:
Ok, since this is not the main test class I changed the code to throw Exceptions as we discussed on the slack.